dpp/src/errors/codes.rs

```diff
@@ -105,6 +105,10 @@ impl ErrorWithCode for BasicError {
             Self::InvalidJsonSchemaRefError { .. } => 1014,
             Self::InconsistentCompoundIndexDataError { .. } => 1021,
 
+            Self::DataContractUniqueIndicesChangedError { .. } => 4016,
+            // TODO  -  they don't have error codes in  https://github.com/dashevo/platform/blob/25ab6d8a38880eaff6ac119126b2ee5991b2a5aa/packages/js-dpp/lib/errors/consensus/codes.js
+            Self::DataContractHaveNewUniqueIndexError { .. } => 0,
```

Comment:
@shumkov @shuplenkov @jawid-h Do you guys know anything about why those errors do not have codes?

Reply:
I would recommend comparing it with v0.23. I believe we fixed it.